apps/web/pages/mmb/booking-detail.js

```diff
@@ -1,11 +1,31 @@
 // @flow
 
 import * as React from 'react';
+import { BookingDetail } from '@kiwicom/margarita-core';
+import { withRouter, type Router } from 'next/router';
+import { StyleSheet } from '@kiwicom/universal-components';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 
 import Layout from '../../components/Layout';
 
-export default () => (
+type Props = {|
+  +router: Router,
+|};
+
+const BookingDetailScreen = (props: Props) => (
   <Layout>
-    <div>TODO:</div>
+    <View style={styles.wrapper}>
+      <BookingDetail bookingId={props.router.query.id} />
+    </View>
   </Layout>
 );
+
+const styles = StyleSheet.create({
+  wrapper: {
+    marginTop: defaultTokens.spaceXSmall,
+    marginHorizontal: '25%',
```

Comment:
I don't know how this is going to look on small screens?

Reply:
And no background color on the web version?

Reply:
Web MMB has white background, so no need to set one. We should probably improve this yes when we have some general wrapper for media queries.

this is how it will look like on iPhone SE: 

![screenshot 2019-02-06 at 12 21 03](https://user-images.githubusercontent.com/7705041/52338097-b7e7ba00-2a09-11e9-9068-53edc42a9a46.png)